tools/property-extractor/property_extractor.py

```
@@ -0,0 +1,230 @@
+import sys
+import os
+import json
+import re
+
+from pathlib import Path
+from file_pair import FilePair
+from tree_sitter import Language, Parser
+
+from parser import build_treesitter_cpp_library, extract_properties_from_file_pair
+from property_bag import PropertyBag
+from transformers import *
```

Ruff (0.8.2)

`from transformers import *` used; unable to detect undefined names

(F403)